diff --git m4/ax_lib_cgal_core.m4 m4/ax_lib_cgal_core.m4
index f3f3192e..c8779c76 100644
--- m4/ax_lib_cgal_core.m4
+++ m4/ax_lib_cgal_core.m4
@@ -91,7 +91,10 @@ for ac_cgal_iterate in $ac_cgal_dirs ; do
                     CGAL::convex_hull_2(points.begin(),points.end(),std::back_inserter(result));
                     //std::cout << result.size() << " points on the convex hull" << std::endl;]
                     ])]
-                    ,[ac_cgal=yes],[ac_cgal=no],[ac_cgal=yes])
+
+                    dnl Third option here is the cross-compiling case
+                    dnl https://www.gnu.org/software/autoconf/manual/autoconf-2.63/html_node/Runtime.html
+                    ,[ac_cgal=yes],[ac_cgal=yes],[ac_cgal=yes])
 
            LDFLAGS="$LDFLAGS_SAVED"
            export LDFLAGS
